lms/static/sass/shared/_alerts_pattern_library_shim.scss

```diff
@@ -60,31 +71,62 @@ $alert-border: 0.0625rem solid $alert-border-grey;
 
 // everything below here gets added specificity pattern-library-shim
 .pattern-library-shim {
-
     // ----------------------------
     // #GENERAL
     // ----------------------------
-
     &.alert {
-        background-color: $alert-background-color;
-        border: $alert-border;
-        border-radius: $alert-border-radius;
-        border-top-left-radius: 0;
-        border-top-right-radius: 0;
-        padding: 1.25rem;
+        background-color: $white;
```

Comment:
Agreed, let's just keep the shim the same. I'll make a ticket to clean up how we will theme the pattern library itself.